esphome/components/pca9685/pca9685_output.cpp

```diff
@@ -69,7 +70,7 @@ void PCA9685Output::setup() {
     this->mark_failed();
     return;
   }
-  delayMicroseconds(500);
+  esphome::delayMicroseconds(500);
```

Comment:
```suggestion
  delayMicroseconds(500);
```

This is in the esphome namespace, so we don't add `esphome::` here